barretenberg/cpp/src/barretenberg/vm2/constraining/flavor.hpp

```diff
@@ -280,7 +280,6 @@ class AvmFlavor {
 
         ProverPolynomials(ProvingKey& proving_key);
 
-        size_t get_polynomial_size() const { return execution_input.size(); }
```

Comment:
Not needed and was giving problems because I removed execution_input (a dummy column).